corehq/apps/sms/api.py

```diff
@@ -35,6 +35,12 @@
 from corehq.form_processor.utils import is_commcarecase
 from corehq.util.datadog.utils import sms_load_counter
 from corehq.util.python_compatibility import soft_assert_type_text
+from corehq.util.soft_assert import soft_assert
+
+_sms_content_is_unicode = soft_assert(
```

Comment:
What do you think of renaming this to `_py3_soft_assert`? As is it sounds like it returns a boolean value, and this can be used to assert anything (the Unicode check happens elsewhere).

Reply:
https://github.com/dimagi/commcare-hq/pull/24571